src/client/graphics/layers/GameTopBar.ts

```diff
@@ -47,12 +45,16 @@ export class GameTopBar extends LitElement implements Layer {
   private updatePopulationIncrease() {
     const player = this.game?.myPlayer();
     if (player === null) return;
-    const popIncreaseRate = player.population() - this._population;
-    if (this.game.ticks() % 5 === 0) {
-      this._popRateIsIncreasing =
-        popIncreaseRate >= this._lastPopulationIncreaseRate;
-      this._lastPopulationIncreaseRate = popIncreaseRate;
+    if (this.game.ticks() % 5 !== 0) {
+      return;
     }
+
+    const currentPop = player.population();
+    const popIncreaseRate = currentPop - this._popLastTick;
+    this._popLastTick = currentPop;
```

Comment:
Something like this might work:
```suggestion
    const popIncreaseRate = this.game.config().populationIncreaseRate(player);
```